tests/test_nodes.py

```diff
@@ -950,6 +947,32 @@ class TestCase:
             expected = case.expected
             pd.testing.assert_frame_equal(expected, actual)
 
+    def test_vertical_distribution_of_building_heights(self):
+        """Test that the function `vertical_distribution_of_building_heights()` returns the correct dataframe."""
+        building_height = pd.Series([0, 5, 5, 6, 7.5, 75])
```

Comment:
Why do we test 5 twice?

Reply:
Making sure that if we input the same value it has identical returns. but it isn't super necessary